### PR TITLE
wasm: Add a page for LibWasm results

### PR DESCRIPTION
**Pull request by alimpfard**

### CODE REVIEW

**Comment by alimpfard** [comment]
toplevel:
image

some random test, expanded:
image

File: wasm/main.js [hunk 1]
```diff
@@ -0,0 +1,393 @@
+"use strict";
+
+(() => {
```
Comment (linusg) [inline wasm/main.js:3]: wait, do we have to copy & paste all of this? :(

Comment (linusg) [reply wasm/main.js:3]: apparently we do.

File: test262/per-file/index.html [hunk 2]
```diff
@@ -59,6 +59,15 @@ <h2>Per-file results</h2>
...
     <script src="../test-config.js"></script>
     <script src="../fetch.js"></script>
+    <script>
+      var InitialPathInTree = "test";
+      var LoadPathsAndModes = [
+        ["per-file-master.json", ""],
+        ["per-file-bytecode-master.json", "bytecode"],
+      ];
+      var generateGitHubURLFromTestPath = (filepath) =>
+        `https://github.com/tc39/test262/tree/main/${filepath}`;
```
[the remark this answers is not in the document]
Comment (alimpfard) [reply test262/per-file/index.html]: yoinked into `window.config`.

**Pull request merged**